api/custom_auth/oauth/serializers.py

```diff
@@ -26,7 +26,9 @@ class Meta:
 
     def create(self, validated_data):
         user = self._get_user()
-        update_last_login(None, user)
+        user_logged_in.send(
+            sender=get_user_model(), request=self.context.get("request"), user=user
```

Comment:
We've already instantiated the UserModel elsewhere in the module so let's use that here rather than calling the function again. 

```suggestion
            sender=UserModel, request=self.context.get("request"), user=user
```